app/src/common/shared/com/igalia/wolvic/ui/widgets/NavigationBarWidget.java

```diff
@@ -944,6 +948,8 @@ private void exitVRVideo() {
         mAttachedWindow.setVisible(true);
         mMediaControlsWidget.setVisible(false);
         mTrayViewModel.setShouldBeVisible(!mAttachedWindow.isFullScreen() && !mAttachedWindow.isKioskMode());
+        mViewModel.setAutoEnteredVRVideo(false);
+        AnimationHelper.fadeIn(mBinding.navigationBarFullscreen.fullScreenModeContainer, AnimationHelper.FADE_ANIMATION_DURATION, null);
```

Comment:
I don't think this fadeIn() is needed?

Reply:
This is actually needed, we won't have the fullscreen navigation bar without this if you open one of the auto-vr projection recognition videos and choose 2D to go back to full-screen playing.